manual/logging/README.md

```diff
@@ -113,78 +113,73 @@ and provide hints about what's going wrong.
 
 ### Logging query latencies
 
-The `QueryLogger` provides clients with the ability to log queries
-executed by the driver, and especially, it allows client to track slow
+The `EnhancedQueryLogger` class provides clients with the ability to log queries
```

Comment:
I was thinking about a blog post actually. I think we will need one for the new mapper features anyway.